core-build/gulp-core-build-sass/src/CSSModules.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+
+import * as postcss from 'postcss';
+import * as cssModules from 'postcss-modules';
+import * as crypto from 'crypto';
+
+export interface ICSSModules {
+  /**
+   * Return a configured postcss plugin that will map class names to a
+   * consistently generated scoped name.
+   */
+  getPlugin: () => postcss.AcceptedPlugin;
+
+  /**
+   * Return the CSS class map that is stored after postcss-modules runs.
+   */
+  getCssJSON: () => Object;
```

Comment:
The type here should be something more specific like `{ [className: string]: string }`, or `Map<string, string>`

Reply:
Thanks for tip. I've modified it to be more specific using an interface (it that generally a good option?)